backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/processing/ImportProcessTemplate.java

```diff
@@ -689,7 +690,21 @@ protected void markImported(final ImportRecordType importRecord)
 
 	protected void afterImport()
 	{
-		// nothing to do here
+		runSQLAfterAllImport();
```

Comment:
pls avoid calling runSQLAfterAllImport from afterImport.
afterImport is meant to be overridden by implementations.

better call runSQLAfterAllImport right near (before/after) afterImport is called (in public final ImportProcessResult run() ).